src/centralserver/admin-service/core/src/test/java/org/niis/xroad/centralserver/restapi/service/CertificationServicesServiceTest.java

```diff
@@ -110,6 +125,27 @@ void getShouldThrowNotFoundException() {
         assertThrows(NotFoundException.class, () -> service.get(ID));
     }
 
+    @Test
+    void addCertificationServiceOcspResponder() throws Exception {
+        var mockOcspResponder = ocspResponder();
+        var mockOcspInfo = ocspInfo();
+        when(ocspResponderConverter.toEntity(mockOcspResponder)).thenReturn(mockOcspInfo);
+        when(ocspInfoRepository.save(mockOcspInfo)).thenReturn(mockOcspInfo);
+        when(ocspResponderConverter.toModel(mockOcspInfo)).thenReturn(mockOcspResponder);
+
+        var result = service.addOcspResponder(mockOcspResponder);
+
+        assertThat(result).isEqualTo(mockOcspResponder);
+        verify(ocspResponderConverter).toEntity(mockOcspResponder);
+        verify(ocspInfoRepository).save(mockOcspInfo);
+        verify(ocspResponderConverter).toModel(mockOcspInfo);
```

Comment:
Correct, the test will fail if all 3 of these methods won't be called, so yes explicit verify is not needed here.
Just thought that this may not be obvious for everyone, but can remove them

Reply:
Method parameters are also verified here, so removing these lines would not be equal.